Introducing myself to the dojo

Offikial opened the pull request:
Introducing myself to the dojo, as part of codeschool’s “Mastering
GitHub” mini-challenge

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Offikial committed:
Introducing myself to the dojo

Introducing myself to the dojo, as part of codeschool’s “Mastering
GitHub” mini-challenge

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,5 @@
+INTRODUCTION
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Offikial committed:
Added my favorite code school path

Added my favorite code school path, in response to pull requests review
comment

Offikial committed:
Adding feature branch change

Offikial committed:
Added new deadly skill

codeschool-kiddo committed:
Added new rule about coffee

Offikial committed:
Corrected new rule

Offikial committed:
Merge pull request #2 from Offikial/new_rules

Too many people are spilling coffee

Offikial committed:
Merge branch 'deadly_skills'

Offikial committed:
Added bad coding practices to the kill list

Offikial committed:
Merge branch 'release_branch_1.0'

Offikial committed:
Added personal grievance, which Fixes #3

Offikial committed:
Merge pull request #4 from Offikial/kill_list

Added personal grievance, which Fixes #3

Offikial committed:
Added link to deadly vipers organization
